lib/active_fedora/file/attributes.rb

```diff
@@ -49,6 +49,11 @@ def create_date
     created && created.first
   end
 
+  def modified_date
+    modified = metadata.attributes["http://fedora.info/definitions/v4/repository#lastModified"]
+    modified && modified.first
```

Comment:
I wouldn't think it would be possible except potentially in "relaxed mode" coming in 4.7.4 as mentioned here: https://wiki.duraspace.org/display/FEDORA4x/How+to+allow+user-updates+to+certain+server+managed+triples .
@mikedurbin is it ever possible for fedora:lastModified to be multivalued?